sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Release History
 
+### 4.3.0b3 (Unreleased)
+
+### Bugs fixed
```

Comment:
Should this be a 4.2.1 hotfix instead? Then if possible unlist/remove 4.2.0.

Reply:
we'll be releasing this in the next beta so no worries on the versioning